pkg/version/version.go

```diff
@@ -42,12 +42,8 @@ func LatestReleaseInDir(releasesDir string) (string, error) {
 		filename := file.Name()
 
 		if !strings.HasPrefix(filename, ReleasesPrefix) {
-			return "", fmt.Errorf(
-				"found non-release prefix ('%s') file '%s' in '%s'",
-				ReleasesPrefix,
-				filename,
-				releasesDir,
-			)
+			// Skipping this file, since it is not a release file
+			continue
```

Comment:
I'm not sure I want to see a warning line for every file that's skipped. in the oss suite we have a single folder with all release YAMLs; for DAP, we have a separate folder for each release that includes the YAML for that release AND a bunch of other files. it would be pretty verbose to have a log line for each other file in the DAP release dirs; if there's utility in being alerted that files are skipped, maybe we could aggregate the filenames and log a single line that lists them all, or just echo that some files were skipped.